templates/v6.yml

```diff
@@ -17,194 +11,73 @@ cache:
     - julia/deps/srccache
     - julia/deps/srccache/*.tar.gz
 
-
 #
 # Julia versions
 #
 
-.julia:0.7:
-  before_script:
-    - apt-get -qq update
-    - apt-get -qq -o=Dpkg::Use-Pty=0 -y install wget ${CI_APT_INSTALL}
-    - >
-      if [ "${CI_RUNNER_EXECUTABLE_ARCH}" == "linux/amd64" ]; then
-        wget -nv -N -P downloads/ https://julialang-s3.julialang.org/bin/linux/x64/0.7/julia-0.7-latest-linux-x86_64.tar.gz
-        FILENAME=julia-0.7-latest-linux-x86_64.tar.gz
-      elif [ "${CI_RUNNER_EXECUTABLE_ARCH}" == "linux/arm64" ]; then
-        wget -nv -N -P downloads/ https://julialang-s3.julialang.org/bin/linux/aarch64/0.7/julia-0.7-latest-linux-aarch64.aarch64.gz
-        FILENAME=julia-0.7-latest-linux-aarch64.aarch64.gz
-      else
-        echo "Unsupported platform:" ${CI_RUNNER_EXECUTABLE_ARCH} >&2
-      fi
-    - tar -C /usr/local -x -z --strip-components=1 -f downloads/julia.tar.gz
-    - julia -e 'using InteractiveUtils; versioninfo()'
-    - JULIA_LLVM_ARGS='--version' julia
-    # https://gitlab.com/gitlab-org/gitlab-runner/issues/327
-    # https://gitlab.com/gitlab-org/gitlab/issues/16343
-    - export JULIA_DEPOT_PATH="$CI_PROJECT_DIR/.julia"
-
 .julia:1.0:
   before_script:
-    - apt-get -qq update
-    - apt-get -qq -o=Dpkg::Use-Pty=0 -y install wget ${CI_APT_INSTALL}
```

Comment:
Why remove this? https://askubuntu.com/a/668859

Reply:
When I was running it I found that it didn't really do much, but I will put it back in the hopes that it saves a few lines.